### ISSUE TITLE
A Princesa e a Ervilha

### ISSUE THREAD

**Issue by vish-blog** [vish-blog/comments#226]
Post de referência: https://www.vish.blog.br/imagens/a-princesa-e-a-ervilha/

**Comment by brunoaugusto** [comment]
Apesar de toda fantasia e bizarrice, a maioria dos contos de fadas até que fazem algum sentido. Não é o caso desse :stuck\_out\_tongue:

**Comment by one-eyed-willy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 20 de junho de 2022](<https://www.vish.blog.br/imagens/a-princesa-e-a-ervilha/#comment_1160636319>):

> Apesar de toda fantasia e bizarrice, a maioria dos contos de fadas até que fazem algum sentido. Não é o caso desse 😛

Sei lá, cara, não lembro de muita coisa desse

**Comment by angellisy** [comment]
Morreu, mas passa bem

**Comment by najarin-the-loremaster** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 20 de junho de 2022](<https://www.vish.blog.br/imagens/a-princesa-e-a-ervilha/#comment_1160669085>):

> Sei lá, cara, não lembro de muita coisa desse

Não, basicamente é só isso mesmo. Numa noite de chuva uma suposta princesa pede abrigo no castelo de um príncipe. Ele se interessa pela moça, mas devido à questão de linhagem, a mãe do rapaz testa a jovem colocando uma ervilha sob 20 colchões e 20 cobertas da cama oferecida. Na manhã seguinte, o relato da noite mal dormida bem como as marcas em suas costas provam a sensibilidade que só uma princesa teria e o casamento é aprovado.
